stub/src/test/proto/helloworld/helloworld.proto

```diff
@@ -0,0 +1,49 @@
+// Copyright 2020 The gRPC Authors
```

Comment:
Is this file being used?
Should it match https://github.com/grpc/grpc-kotlin/blob/master/examples/src/main/proto/hello_world.proto?

Reply:
This is being used, yes, and I do not feel it needs to match since it's for testing only.